examples/two-step-transfer/Cargo.toml

```diff
@@ -10,12 +10,10 @@ repository = "https://github.com/concordium/concordium-rust-smart-contracts"
 readme = "../README.md"
 
 [features]
-default = ["std"]
+default = ["std", "concordium-quickcheck"]
 
 std = ["concordium-std/std"]
-
-[dependencies]
-byteorder = "1.3"
+concordium-quickcheck = ["concordium-std/concordium-quickcheck", "concordium-contracts-common/concordium-quickcheck"]
```

Comment:
Do we need this as a feature in the contract? Maybe it should just have `"concordium-quickcheck"` enabled directly on the dependency.

Reply:
I think you're right, we just want to decorate a test with `#[concordium_quickcheck]` macro. I don't think `concordium-quickcheck` should be a feature here.

I fixed it now.